.binder/postBuild

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+# update pip
+pip install --upgrade pip
+
+# display installed versions
+python -V
+pip list
+
+# install NLTL Punkt tokenizer
+python -m nltk.downloader punkt
+
+# install SpaCy language model
+python -m spacy download de_dep_news_trf
```

Comment:
maybe this is not a good idea, because the model is pretty huge. it increases the size of the docker image significantly and slows down the startup quiet a bit.